### PR TITLE
Schema 2023-08

### PR DESCRIPTION
**Pull request by selimnairb**
Schema version 2023-08 includes changes described in the following issues:

- #7 
- #8

### CODE REVIEW

**Review by glabmoris — APPROVED** [review]

**Review by brian-r-calder — APPROVED** [review]
Looks good to me, and what we discussed.

**Review by georgiannazelenak — APPROVED** [review]
The changes all look as I would expect.  Thank you!

**Comment by selimnairb** [comment]
@glabmoris @brian-r-calder @georgiannazelenak Thanks for the review. I will merge and publish to PyPI.

File: docs/IHO/b12_v3_1_0_example-2023-03.json [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        }
+      ],
+      "soundSpeedDocumented": true,
+      "positionOffsetsDocumented": true,
+      "dataProcessed": true,
+      "contributorComments": "On 2022-03-08, at 20:30 UTC, the echo sounder lost bottom tracking after the vessel crossed another vessel's wake."
+    },
+    "processing": [
+      {
+        "type": "soundSpeed",
```
Comment (giumas) [inline docs/IHO/b12_v3_1_0_example-2023-03.json:56]: Shouldn't the first 's' of "soundSpeed" be upper case?

File: tests/integration/validate_cmd.sh
Comment (giumas) [inline tests/integration/validate_cmd.sh:1]: Given the applied renaming from cmd to command in other places, should this file also be renamed (validate_command.sh)?

File: csbschema/__init__.py [hunk 2]
```diff
@@ -4,19 +4,30 @@
 from csbschema import validators
 
 
-__version__ = '1.0.5'
-
+__version__ = '1.1.0'
 
+# Current versions
+B12_VERSION_3_1_0_2023_08 = '3.1.0-2023-08'
+B12_VERSION_3_0_0_2023_08 = '3.0.0-2023-08'
```
Comment (giumas) [inline csbschema/__init__.py:11]: What is the versioning approach that is enforced? Where is the "-2023-08" coming from?

File: csbschema/data/CSB-schema-3_2_0-BETA.json [hunk 3]
```diff
@@ -492,6 +576,33 @@
         }
       }
     },
+    "VerticalOffsetAnalysisOffset": {
```
Comment (giumas) [inline csbschema/data/CSB-schema-3_2_0-BETA.json:579]: 'Offset' seems duplicated. Does the naming follow a specific logic? A meaningful name reduces the need for documentation.

File: docs/IHO/b12_v3_1_0_example-2023-03.json [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+          40.941789,
+          19.105552
+        ]
+      },
+      "properties": {
+        "depth": 16.7,
+        "time": "2016-03-03T18:41:52.2342z"
```
Comment (giumas) [inline docs/IHO/b12_v3_1_0_example-2023-03.json:140]: Is the mix of uppercase 'T' and lowercase 'z' intended?

File: docs/IHO/b12_v3_1_0_example-2023-03.json (hunk 1 above)
Comment (selimnairb) [reply docs/IHO/b12_v3_1_0_example-2023-03.json:56]: Yes, this is a bug in the 3.1.0 example files. Working on a fix and will commit. Thanks.

Comment (selimnairb) [reply docs/IHO/b12_v3_1_0_example-2023-03.json:56]: See #11

**Pull request merged**